### ISSUE TITLE
filenameForDownload is not setting the document name

### ISSUE THREAD

**Issue by Penchal07** [stephanrauh/ngx-extended-pdf-viewer#752]
Im downloading the document from the blob with document name,

API:
```typescript 
 if (excelBlob != null)
   {
          excelBlob.Position = 0;
          return File(excelBlob, "application/octet-stream", Path.Combine(blobName, ".pdf"));
      }
```
filenameForDownload is not working to set the document name. Below is the code Im using,

```html
  <ngx-extended-pdf-viewer [src]="pdfSrc" useBrowserLocale="false" height="690px"
        [filenameForDownload]="filenameForDownload" [enableDragAndDrop]="false" [enablePrint]="false"
        [showOpenFileButton]="false" [showBookmarkButton]="false" [textLayer]="true" [handTool]="false"
        [startTabindex]="100" [showSidebarButton]="true"  [showDownloadButton]="false" [showRotateButton]="false" (pdfLoaded)="padfLoaded($event)">
      </ngx-extended-pdf-viewer>
```
Component:

```typescript
 loadk1PDF() {
    this.filenameForDownload = "ScheduleK1.pdf"
    this.pdfSrc = new Blob([res], { type: res.type });
.................
....................
}
```

![image](https://user-images.githubusercontent.com/83951393/117639040-1d0a5700-b1a1-11eb-8dc1-6b269005de15.png)

Then I'm trying to view the document properties, there it showing name as **document.pdf**

**Comment by Penchal07** [comment]
Sorry, I was not working. Can you suggest me how I can set the pdf name in the properties window?

https://user-images.githubusercontent.com/83951393/117639040-1d0a5700-b1a1-11eb-8dc1-6b269005de15.png

**Comment by stephanrauh** [comment]
Congratulations - you've found not only one, but two bugs!

I've just published version 9.0.0-alpha6. It should solve your issue.

**Comment by Penchal07** [comment]
> Congratulations - you've found not only one, but two bugs!
> 
> I've just published version 9.0.0-alpha6. It should solve your issue.

Thank so much buddy!!! It resolved the issue.
